inference-engine/tests_deprecated/unit/engines/gna/I8_quantisation_test.cpp

```diff
@@ -79,19 +79,6 @@ TEST_F(I8QuantisationTest, inputPrecisionIs16Bits){
     ASSERT_EQ(inputLayer->precision, Precision::I16);
 }
 
-TEST_F(I8QuantisationTest, failIfFCDimensionIs1){
```

Comment:
Consider to add a test, or modify this one instead removing.

Reply:
Modified the test. Unfortunately it's impossible to add a test for 1d tensors on ngraph since ngraph rejects configurations with 1d tensors.

Reply:
@elilobanova do you know the reason for that? Maybe ask someone from ngraph team?